test/src/commands/create/create_subcommand_test.dart

```diff
@@ -285,7 +286,61 @@ Run "runner help" to see global options.''';
                 that: isA<Directory>().having(
                   (d) => d.path,
                   'path',
-                  'test_dir/test_project',
+                  path.join('test_dir', 'test_project'),
```

Comment:
There are probably other places where this exists `/`. 

Without `path.join` the tests are broken on Windows.